lib/sles4sap/console_redirection.pm

```diff
@@ -37,6 +37,7 @@ our @EXPORT = qw(
   connect_target_to_serial
   disconnect_target_from_serial
   check_serial_redirection
+  $ssh_opt
 );
 
 my $ssh_opt = join(' ',
```

Comment:
is it ok to use `my` for variable that are exported? (should not be `our`)?

Reply:
I removed this export completely.